### PR TITLE
[JENKINS-73038] Adapt plugin to allow user with Overall/Manage to configure global settings

### PR DESCRIPTION
**Pull request by mikecirioli**
Adapt plugin to allow user with Overall/Manage to configure global settings under `Manage Jenkins` --> `System`

Jenkins Issue -> https://issues.jenkins.io/browse/JENKINS-73038

### Your checklist for this pull request

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side) and not your master branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or in [Jenkins JIRA](https://issues.jenkins-ci.org)
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Did you provide a test-case? That demonstrates feature works or fixes the issue.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>4.80</version>
+    <version>4.81</version>
```
Comment (LyroStedman) [inline pom.xml:8]: Isn't it risking conflict with the automatic dependency PR  https://github.com/jenkinsci/bitbucket-branch-source-plugin/pull/844 ?

File: pom.xml [hunk 2]
```diff
@@ -30,6 +30,7 @@
     <gitHubRepo>jenkinsci/bitbucket-branch-source-plugin</gitHubRepo>
     <jenkins.version>2.401.3</jenkins.version>
     <hpi.compatibleSinceVersion>2.0</hpi.compatibleSinceVersion>
+    <useBeta>true</useBeta> <!-- Jenkins.MANAGE -->
```
Comment (LyroStedman) [inline pom.xml:33]: Is it supposed to stay or was it to test ?

File: pom.xml (hunk 1 above)
Comment (mikecirioli) [reply pom.xml:8]: if so i will resolve it

**Comment by mikecirioli** [comment]
@LyroStedman / @basil  / @rsandell could i get an additional review when you have a minute?

**Review by basil — APPROVED** [review]

**Review by amuniz — APPROVED** [review]

File: pom.xml (hunk 2 above)
Comment (LyroStedman) [reply pom.xml:33]: OK, I read badly this conf, I thought it was forcing the plugin in beta mode -_-
My bad, sorry.

Comment (mikecirioli) [reply pom.xml:33]: no worries, i've also started [JENKINS-73089](https://issues.jenkins.io/browse/JENKINS-73089) to address moving the `Overall/Manage` permission out of `beta` status

**Review by joseblas — APPROVED** [review]
hey @mikecirioli  how are you doing :)

**Pull request merged**